README.md

```diff
@@ -11,157 +18,79 @@ Client:
 
 Server:
 
-- Node.js,
+- Node,
 - Express.js
 
 Database:
 
 - MongoDB
 
-Tools:
-
-- Webpack,
-- Eslint (custom rules extends Airbnb),
-- Prettier,
-- Husky (pre-commits hooks).
-- Docker
-
-### Seeding database for development
-
-At first run of the app, log in and check your user id in the database under eoc/users directory. Put that user ID, generated by mongo to .env file under USER_ID variable and save the file.
-Then simply run `npm run seed`.
-Seeding is not working? Probably USER_ID in your database is not matching with USER_ID in your .env file. Be aware that if you clean up the whole database with users collection, your USER_ID delivered by mongo will be different compared to the previous session. **TIP** When playing with data, always clean all the collections omitting users collection, so you don't have to deal with `USER_ID` configuration over and over again.
-
-### Google Client ID and Client Secret Key
-
-Before running the app remember to add the Google client id key, Client secret key and Express session key to .env file.
+All of the used technologies you can find in package.json files.
 
-### Webpack Development mode
+## Installation
 
-In the development mode, we will have 2 servers running. The front end code will be served by the [webpack dev server](https://webpack.js.org/configuration/dev-server/) which helps with hot and live reloading. The server side Express code will be served by a node server using [nodemon](https://nodemon.io/) which helps in automatically restarting the server whenever server side code changes.
+Please remember first to create .env file in the root directory of the project and add necessary .env keys that you can find in the .env-example file.
 
-### Webpack Production mode
-
-In the production mode, we will have only 1 server running. All the client side code will be bundled into static files using webpack and it will be served by the Node.js/Express application.
-
-## Quick Start
-
-```bash
+```
 # Clone the repository
```

Comment:
Please split this into a numbered list of steps and their shell commands.

For example:
```md
1. Clone the repository
```sh
git clone https://github.com/sandstreamdev/eoc.git
```
2. Go inside the directory...
```